src/components/policy/test/sql_pt_representation_test.cc

```diff
@@ -996,6 +997,7 @@ const std::string kAppStorageFolder = "storage";
 }
 
 TEST(SQLPTRepresentationTest3, Init_InitNewDataBase_ExpectResultSuccess) {
+  file_system::CreateDirectory(kAppStorageFolder);
```

Comment:
@jacobkeeler in case of exception the directory will never be removed.
I propose to use one of the following approaches
1. Either wrpat `Create/RemoveDirectory`  with some RAII class
2. Or use test fixture and perform creationd/deletion in the `setup/teardown` method